src/main/java/org/ow2/proactive/catalog/dto/CatalogObjectMetadata.java

```diff
@@ -50,6 +50,9 @@
 @EqualsAndHashCode(callSuper = false)
 public class CatalogObjectMetadata extends ResourceSupport {
 
+    @JsonProperty("commit_time_raw")
+    protected final String commitTimeRaw;
```

Comment:
the current commit_time is the commit time but displayed in a human-understandable way. I created  commit_time_raw which is the same time, but as a long (it's the ID of the revision)